apps/settings/js/modules/bluetooth/bluetooth_adapter_manager.js

```diff
@@ -0,0 +1,113 @@
+/* global console */
```

Comment:
Let's add `console` to `apps/settings/.jshintrc`, then we don't have to declare `console` anymore.